server/py/services/api/crud/alert_activation.py

```diff
@@ -13,10 +13,62 @@
 # limitations under the License.
 #
 
+import sqlalchemy.orm
+
+import mlrun.common.schemas.alert
 import mlrun.utils.singleton
 
+import framework.utils.singletons.db
+
 
 class AlertActivation(
     metaclass=mlrun.utils.singleton.Singleton,
 ):
-    pass
+    def store_alert_activation(
+        self,
+        session: sqlalchemy.orm.Session,
+        alert_data: mlrun.common.schemas.AlertConfig,
+        event_data: mlrun.common.schemas.Event,
+    ):
+        notifications_states = self._prepare_notifications_states(alert_data)
+
+        framework.utils.singletons.db.get_db().store_alert_activation(
+            session, alert_data, event_data, notifications_states
+        )
+
+    def list_alerts_activations(
+        self,
+        session: sqlalchemy.orm.Session,
+        project: str = "",
+    ) -> list[mlrun.common.schemas.AlertActivation]:
+        # TODO: add filters
+        project = project or mlrun.mlconf.default_project
+        return framework.utils.singletons.db.get_db().list_alerts_activations(
+            session, project
+        )
```

Comment:
We don't really need this now (as unit tests aren't applicable), so I think we can leave it for next PR